src/components/Summary/Summary.jsx

```diff
@@ -0,0 +1,71 @@
+import { Suspense, lazy, useState, useEffect } from "react";
+import { FiMinus } from "react-icons/fi";
+
+import IconButton from "../shared/Button/IconButton";
+import SummarySkeleton from "./SummarySkeleton";
+import generateResponse from "../../utils/api";
```

Comment:
```suggestion
import SummarySkeleton from "./SummarySkeleton";

import generateResponse from "../../utils/api";
```

컴포넌트와 utils의 import문을 분리해 주면 좋을 것 같습니다!